modules/serverless-channel-configmap.adoc

```diff
@@ -0,0 +1,37 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-channels.adoc
+
+[id="serverless-channel-configmap_{context}"]
+= Default ConfigMap
+
+The default channel configuration is specified in the ConfigMap named `default-ch-webhook` in the `knative-eventing` namespace. This ConfigMap may specify a cluster-wide default channel, as well as namespace-specific channel implementations.
+Namespace-specific defaults override cluster-wide defaults created in the specified namespace if both configurations define the same fields.
+
+.Example default ConfigMap
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-ch-webhook
+  namespace: knative-eventing
+data:
+  default-ch-config: |
+    clusterDefault: <1>
+      apiVersion: messaging.knative.dev/v1beta1
+      kind: InMemoryChannel <2>
+    namespaceDefaults: <3>
+      example-namespace:
+        apiVersion: messaging.knative.dev/v1beta1
+        kind: InMemoryChannel <4>
+        spec:
+          numPartitions: 2
```

Comment:
InMemoryChannel doesn't have numPartitions and replicationFactor properties AFIAK  ( I understand this example comes originally from KafkaChannel, which does have these properties, but we don't want to show *that* )  

Maybe we can just drop the spec:?  (I don't think it would really be a problem if the clusterDefault and namespaceDefaults were exactly the same here... )

Also, I am not sure if users should actually be modifying the ConfigMap , or is that reserved for the Operator 

IMHO since we don't actually have a supported use case for changing the channel yet, I don't think we should have this part at all.

Reply:
That makes sense - removed the configmap and any references to changing the type - does this look better now? :slightly_smiling_face: